taskcluster/ci/ui-test/kind.yml

```diff
@@ -2,23 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 ---
-loader: focus_android_taskgraph.loader.multi_dep:loader
+loader: taskgraph.loader.transform:loader
 
 transforms:
-    - focus_android_taskgraph.transforms.multi_dep:transforms
     - taskgraph.transforms.job:transforms
     - taskgraph.transforms.task:transforms
 
 kind-dependencies:
     - build
-    - toolchain
```

Comment:
Once you add the signing kind, I think you'll see why the the multi_dep transform is needed, unless there's some other way to have one task pending until another task completes. Also, if you look at fenix, the dependency should change once you add the signing kind; this task will no longer be primarily dependent on the build to finish (that will be the signing kind), and this in turn will depend on the signing kind to finish. The reason why I didn't set this kind up is because for the first pr, the `focusDebug` is signed by default with a debug key, and that's all that's needed for the ui-test kind.